README.md

```diff
@@ -660,6 +660,27 @@ terraform state rm aws_elasticsearch_domain.index
 … to update the Terraform state so that it reflects the deletion of the
 Elasticsearch domain. Now running `make deploy` should succeed. 
 
+
+## `NoCredentialProviders` while running `make deploy`
+```
+Failed to save state: failed to upload state: NoCredentialProviders: no valid providers in chain. 
+…
+The error shown above has prevented Terraform from writing the updated state
+to the configured backend. To allow for recovery, the state has been written
+to the file "errored.tfstate" in the current working directory.
+
+Running "terraform apply" again at this point will create a forked state,
+making it harder to recover.
+```  
+This happens when the auth token expires mid flight of `terraform apply`,
+confirm auth by running `_preauth`. Then run …
+```
+$ cd terraform
```

Comment:
Nit 1. I would drop the `$` so people can copy paste the line. 
Nit 2. @hannes-ucsc recommended to me to use `()` subshells  in another PR, i would suggest it here.

```
(cd terraform && terraform state push errored.tfstate)
```